SmartEVSE-3/src/evse.cpp

```diff
@@ -2110,6 +2164,210 @@ uint8_t PollEVNode = NR_EVSES;
 
 }
 
+#ifndef MQTT_DISABLED
+void mqtt_receive_callback(const String &topic, const String &payload)
+{
+    if (topic == MQTTprefix + "/Set/Mode")
+    {
```

Comment:
General layout remark: please keep opening { on the same line as the if statement....
So: if (topic == MQTTprefix) {
            //something 
      }

this saves (almost) empty lines, which fits more lines of code on your screen.....

Reply:
Normally I'd agree, but while working on this project it really helped to have the extra tidy-ness it brings while trying to understand the somehow somewhat convoluted logic. I suppose it's a matter of opinion, so I'll make it in line with the rest of the code.